ncm-metaconfig/src/test/perl/service-opennebula.t

```diff
@@ -0,0 +1,14 @@
+use Test::Quattor::ProfileCache qw(set_json_typed get_json_typed);
+BEGIN {
+    set_json_typed()
+}
+
+use Test::More;
+use Test::Quattor::TextRender::Metaconfig;
+
+my $u = Test::Quattor::TextRender::Metaconfig->new(
+        service => 'opennebula',
```

Comment:
@stdweird , @kwaegema  sorry I forgot to apply this change.. :disappointed:  done!